build.boot

```diff
@@ -91,8 +91,10 @@ Type \"Yes\" to confirm."
   (let [env (System/getenv "WOMBATS_ENV")]
     (if (or (= env "")
             (nil? env))
-      "dev"
-      env)))
+      (do (System/setProperty "APP_ENV" "dev")
```

Comment:
This is the same as the way it was implemented before - I figured that the system properties carried over and the environment variables did not, but it makes sense that they wouldn't. What would you suggest?